intercom/traits/incrementable_attributes.py

```diff
@@ -5,4 +5,6 @@ class IncrementableAttributes(object):
 
     def increment(self, key, value=1):
         existing_value = self.custom_attributes.get(key, 0)
+        if existing_value is None:
+            existing_value = 0
```

Comment:
Alternative:
```python
existing_value = self.custom_attributes.get(key) or 0
```

Reply:
I prefer the explicitness of checking is `None`.